site/content/integrate/admin-guide/_index.md

```diff
@@ -71,7 +71,7 @@ We currently don't support the ability to attach files to a post made by an inte
 
 ## Where can I find existing integrations?
 
-[Visit our app directory](https://about.mattermost.com/default-app-directory/) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and Bitbucket, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
+[Visit our app directory](https://mattermost.com/marketplace/#privateApps) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and Bitbucket, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
```

Comment:
```suggestion
[Visit our app directory](https://mattermost.com/marketplace/) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and Bitbucket, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
```

Reply:
I'm not sure where the suggested URL originates - I've checked and it links to https://mattermost.com/marketplace/. If you're able to share the source, I can confirm whether we use that one or not. Thank you!

Reply:
I just tried it on the live page https://developers.mattermost.com/integrate/admin-guide/ and it redirects to that URL with the #privateApps towards the end. But if you feel that we don't need this one in the link and just marketplace is the right one, we can commit your changes and proceed. Anyways both land up on the Market Place page only.

Reply:
That's very interesting. I'm not sure why that is. @mickmister, do you happen to know why this would be appended to the URL and whether it's required?

Reply:
I'm not sure how it originated, but we can definitely have the URL without the `#privateApps` part